app/src/main/kotlin/com/wire/android/ui/userprofile/other/OtherUserProfileScreen.kt

```diff
@@ -371,43 +373,54 @@ private fun Content(
     getOtherUserClients: () -> Unit,
     onDeviceClick: (Device) -> Unit
 ) {
-    Crossfade(targetState = tabItems to state) { (tabItems, state) ->
-        when {
-            state.isDataLoading || state.botService != null -> Box {} // no content visible while loading
-            state.connectionState == ConnectionState.ACCEPTED ->
-                CompositionLocalProvider(LocalOverscrollConfiguration provides null) {
-                    HorizontalPager(
-                        modifier = Modifier.fillMaxSize(),
-                        state = pagerState,
-                        count = tabItems.size
-                    ) { pageIndex ->
-                        when (val tabItem = tabItems[pageIndex]) {
-                            OtherUserProfileTabItem.DETAILS ->
-                                OtherUserProfileDetails(state, otherUserProfileScreenState, lazyListStates[tabItem]!!)
+    Column {
+        OtherUserConnectionStatusInfo(state.connectionState, state.membership)
```

Comment:
Now, with extracting `OtherUserConnectionStatusInfo` out of the `Crossfade` we no longer have a "fade" animation when user is connecting and the content changes from "not connected" to "connected" - if that's fine then it can be accepted, but I always cry inside a little when someone decreases the overall user experience level 😄